### ISSUE TITLE
document cppcheck install for OSX

### ISSUE THREAD

**Issue by mr-c** [dib-lab/khmer#777]
Source from http://sourceforge.net/projects/cppcheck/files/cppcheck/1.68/ (the `.tar.gz` file)
```
make SRCDIR=build CFGDIR=/usr/share/cppcheck/cfg
sudo make SRCDIR=build CFGDIR=/usr/share/cppcheck/cfg install
sudo mkdir -p /usr/share/cppcheck/cfg
sudo cp cfg/* /usr/share/cppcheck/cfg/
```

**Comment by elmbeech** [comment]
I am using OS X with mac port (https://www.macports.org/). I will write document how to install cppchecker for users who run their mac with the mac port distro.

**Comment by safay** [comment]
@mr-c @elmbeech Please note cppcheck is also installable by homebrew; can you include that too?
`brew install cppcheck`

**Comment by elmbeech** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Is the Copyright year up to date?

**Comment by elmbeech** [comment]
@ safay 
Can you drop me a line how to install pip 2.7 with homebrew?

**Comment by elmbeech** [comment]
There is no copyright notification in the getting-started.rst file. 
Should I add one?

**Comment by mr-c** [comment]
Fixed in #945
